### ISSUE TITLE
Define and encourage standards GitHub Topics for non-official repositories related to Rocket.Chat

### ISSUE THREAD

**Issue by fititnt** [RocketChat/Rocket.Chat#10032]
**TL;DR: Rocket.Chat is a powerhouse on GitHub, but it's adapters/bridges/integrations could have some basic marketing help on how to promote they using [GitHub Topics](https://help.github.com/articles/about-topics/); for example, HuBot uses [hubot-adapter](https://github.com/topics/hubot-adapter) topic, but most repositories related to Rocket.Chat depends on less efficient and unreliable string search**

----

In special because of the issue **Rocket.Chat Apps #6890** and because of [this tweet asking feedback by chatbot people](https://twitter.com/RocketChat/status/966770860777500672), even if at some point someone decide have some marketplace inside the rocketchat app, would be nice make easier for developers find extensions on GitHub that have some sort of integration with Rocket.Chat. 

For example, is common even for good developers struggle with some new software they are trying to integrate. Promote ways to find these people, and be found, is a win-win, even if these repositories still not mature to be part of some market place. 

My proposal here is the Rocket.Chat developer and marketing working teams define some more standard ways to use [GitHub Topics](https://help.github.com/articles/about-topics/) as one way to promote a healthy collaborations between developers who want make integrations with rocket chat by at least proposing some topics.

----

## Printscreens about searches on GitHub

As reference, I will post here as reference of current state.

### [Search by "rocketchat" string](https://github.com/search?utf8=%E2%9C%93&q=rocketchat):

![captura de tela de 2018-03-06 13-11-13](https://user-images.githubusercontent.com/812299/37044315-e5e51ff0-2141-11e8-9cc2-39466f12fae4.png)

### [Search by "rocketchat-adapter" string](https://github.com/search?utf8=✓&q=rocketchat-adapter)

Tip: none have any Topic. Also only one has stars.

![captura de tela de 2018-03-06 13-34-38](https://user-images.githubusercontent.com/812299/37044921-7c0bfa8e-2143-11e8-9fa6-8f5a202edf51.png)

### [Search by "rocketchat-integration"](https://github.com/search?utf8=✓&q=rocketchat-integration)

Tip: only one have topics, but the topic is "rocketchat" itself. Even if no one will go for a trademark, still good encourage some standards.

![captura de tela de 2018-03-06 13-34-38](https://user-images.githubusercontent.com/812299/37045002-b5c243d2-2143-11e8-81c7-43d26f4bc94a.png)

## Motivation

A step by step motivation. The only way I discovered the repository [jacted/speed-monitor](https://github.com/jacted/speed-monitor) was because twitter algorithm decided to show me this

![captura de tela de 2018-03-06 13-07-16](https://user-images.githubusercontent.com/812299/37043650-f1147382-213f-11e8-8255-49f2e5834953.png)

The repository is this. See chat there are no topics. Description "Website performance reports for Slack, Rocket.chat & Mattermost using performance.timing"

![captura de tela de 2018-03-06 13-13-33](https://user-images.githubusercontent.com/812299/37043728-36c03790-2140-11e8-8f3e-b18f8d9858ed.png)

Problem: this repository cannot be found using GitHub search function for "rocketchat" even if we sort by stars instead of "best match" (default). Also using exactly  "Rocket.chat" does not return this repository on "best match" and only by "most stars" will return.

Note that most extensions for chat platforms tends to have slow amount of stars. So we provably missing a lot out there. IMHO a good promotion of this standards way could be even more efficient than just at some point sponsor hackathons, since the RC already have a good usage on it's field.

**Comment by MartinSchoeler** [comment]
@fititnt Could you reopen this in https://forums.rocket.chat/ ? We aim to keep github for bugs and feature request only. Thanks!

**Comment by fititnt** [comment]
@MartinSchoeler sure! Good to know about this. Done at forums  **_feedback_** category. As soon as approved will update here with the direct link.

**Comment by geekgonecrazy** [comment]
Not sure why it got caught up needing approved.  But i've approved:

https://forums.rocket.chat/t/define-and-encourage-standards-github-topics-for-non-official-repositories-related-to-rocket-chat/316

**Comment by fititnt** [comment]
Thanks for updating with the link, @geekgonecrazy. For who is interested, we discuss there.

---

> Not sure why it got caught up needing approved. But i've approved:

I'm my opinion was a good default that the discuss platform required approval. Think that my account had been created and email-confirmed for something like less than 3 min and I copy pasted this full post with links for remote domain (github) and with images (github CDN). So this is not a bug, but a feature.
